lib/ui/page/home/page/my_profile/view.dart

```diff
@@ -813,7 +819,126 @@ Widget _media(BuildContext context, MyProfileController c) {
           ),
         );
       }),
-      SizedBox(height: 8),
+
+      // Voice processing widget
+      Column(
+        children: [
+          if (PlatformUtils.isDesktop && !PlatformUtils.isWeb) ...[
+            const SizedBox(height: 20),
+            LineDivider('label_voice_processing'.l10n),
+            const SizedBox(height: 16),
+            Obx(() {
+              // False by default
```

Comment:
Comments should have trailing dot. https://dart.dev/effective-dart/documentation#do-format-comments-like-sentences